### ISSUE TITLE
Error in 2.1.0 index.js Unable to resolve some modules 'TogglTextFieldcke'

### ISSUE THREAD

**Issue by nickvirtue** [erikras/redux-form-material-ui#7]
After npm updating to 2.1.0 we have the following error:

```
Unable to resolve some modules:

  "./TogglTextFieldcke" in ...../node_modules/redux-form-material-ui/lib/index.js
(web.browser)
```

Reverting to 2.0.0 solves issue. 

Looks like it's supposed to be Toggle but possibly messed up in transpiling.

**Comment by erikras** [comment]
Saw that too. Very weird npm/IDE error.

**Comment by erikras** [comment]
Fixed in `v2.1.1`. Apologies.
